spec/spec_helper.rb

```diff
@@ -122,3 +122,15 @@
     with.library :rails
   end
 end
+
+# Omniauth
+OmniAuth.config.test_mode = true
```

Comment:
To keep our spec_helper.rb from getting too cluttered, you can move this to its own file, i.e. `spec/support/omniauth_helper.rb`.

Reply:
I see that you created an empty `spec/support/omniauth_helper_spec.rb`, but left the code in `spec_helper`. Also, that file can just be called `spec/support/omniauth_helper.rb`, because it's a helper file rather than a spec file.